vowpalwabbit/constant.h

```diff
@@ -34,6 +34,7 @@ constexpr unsigned char node_id_namespace = 136;                   // this is \x
 constexpr unsigned char baseline_enabled_message_namespace = 137;  // this is \x89
 constexpr unsigned char ccb_slot_namespace = 139;
 constexpr unsigned char ccb_id_namespace = 140;
+constexpr unsigned char ldf_namespace = 141;
```

Comment:
I'm guessing this should be excluded from AutoML too?, https://github.com/VowpalWabbit/vowpal_wabbit/blob/912fbd2765f67aea2675352b5aefd42dcc4b5f52/vowpalwabbit/reductions/automl.h#L41

Reply:
Actually this might be below cb_adf so there might not be a need.